ci-operator/config/devfile/devworkspace-operator/devfile-devworkspace-operator-main__v7.yaml

```diff
@@ -49,8 +49,8 @@ tests:
       cli: latest
       commands: |
         echo -e "[INFO] Test script content:"
-        cat .ci/oci-devworkspace-happy-path.sh
-        /bin/bash .ci/oci-devworkspace-happy-path.sh
+        cat .ci/oci-dashboard-happy-path.sh
+        /bin/bash .ci/ooci-dashboard-happy-path.sh
```

Comment:
Is this rename intentional? It looks to be modifying the devworkspace happy path. Also, there's a typo on line 53 -- `ooci`.